pandora/impl/pytorch/utils.py

```diff
@@ -32,10 +32,10 @@ def init_rnn(rnn):
 
     else:
         for layer in range(rnn.num_layers):
-            init.xavier_uniform(getattr(rnn, f'weight_hh_l{layer}'))
-            init.xavier_uniform(getattr(rnn, f'weight_ih_l{layer}'))
-            init.constant(getattr(rnn, f'bias_hh_l{layer}'), 0.)
-            init.constant(getattr(rnn, f'bias_ih_l{layer}'), 0.)
+            init.xavier_uniform(getattr(rnn, 'weight_hh_l{layer}'.format(layer=layer)))
```

Comment:
I'd prefer if we followed the PEP8 guideline of keeping lines to a maximum of 80 characters (at least I always do this for the code for which I am responsible). It improves readability a lot.

Reply:
More than fine with that, maybe it's time to put some kind of lint tool here ?

Reply:
I am fine with or without linter. I always have a PEP linter built-in in my editor, so usually I can clean up before commit. Since this is more about readability than avoiding bug-prone code, I'd rather leave it up to committer.

Reply:
Hey !
Actually regarding line length, I am more a 120-length kind of guy. I find 80 to be really low when you are in a deep recursion. Would you agree with a 120 ?